core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -377,11 +379,11 @@ public T maxTraceEvents(int maxTraceEvents) {
   }
 
   /**
-   * Override the default stats implementation.
+   * Sets a {@link ClientInterceptor} that overrides the default stats interceptor implementation.
    */
   @VisibleForTesting
-  protected final T overrideCensusStatsModule(CensusStatsModule censusStats) {
-    this.censusStatsOverride = censusStats;
+  protected final T setCensusStatsInterceptor(ClientInterceptor statsInterceptor) {
```

Comment:
I think we need to consider just adding an interceptor like normal, instead of having a special method for it. We would no longer need testing's `TestingAccessor`.

Reply:
Changed tests to apply census interceptor/stream tracer factory as normal ones. This eliminates the special methods on channel/server builder for overriding census interceptor/stream tracer factory. With this approach, we would still need accessors for disabling default census interceptor/stream tracer factory.